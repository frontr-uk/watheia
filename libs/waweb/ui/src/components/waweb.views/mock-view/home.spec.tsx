import { render } from '@testing-library/react';

import HomeView from './home';

describe('HomeView', () => {
  it('should render successfully', () => {
    const { container } = render(<HomeView />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });
});
