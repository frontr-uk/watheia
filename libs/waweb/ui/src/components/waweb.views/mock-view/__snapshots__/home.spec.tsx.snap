// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView should render successfully 1`] = `
<div>
  <main>
    <div
      class="pt-10 sm:pt-16 lg:pt-8 lg:pb-14"
    >
      <div
        class="mx-auto max-w-7xl lg:px-8"
      >
        <div
          class="lg:grid lg:grid-cols-2 lg:gap-8"
        >
          <div
            class="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 sm:text-center lg:px-0 lg:text-left lg:flex lg:items-center"
          >
            <div
              class="lg:py-24"
            >
              <a
                class="inline-flex items-center text-white bg-black rounded-full p-1 pr-2 sm:text-base lg:text-sm xl:text-base hover:text-gray-200"
                href="#"
              >
                <span
                  class="px-3 py-0.5 text-white text-xs font-semibold leading-5 uppercase tracking-wide bg-gradient-to-r from-teal-500 to-cyan-600 rounded-full"
                >
                  We're hiring
                </span>
                <span
                  class="ml-4 text-sm"
                >
                  Visit our careers page
                </span>
                <svg
                  aria-hidden="true"
                  class="ml-2 w-5 h-5 text-gray-500"
                  viewBox="0 0 18 18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title>
                    S ChevronRight 18 N
                  </title>
                  <path
                    d="M12 9a.994.994 0 0 1-.292.705l-3.992 3.99a1 1 0 1 1-1.436-1.387l.025-.024L9.591 9 6.305 5.715A1 1 0 0 1 7.69 4.28l.025.025 3.991 3.99A.994.994 0 0 1 12 9Z"
                    style="fill: hsl(var(--bc));"
                  />
                </svg>
              </a>
              <h1
                class="mt-4 text-4xl tracking-tight font-extrabold text-white sm:mt-5 sm:text-6xl lg:mt-6 xl:text-6xl"
              >
                <span
                  class="block"
                >
                  A better way to
                </span>
                <span
                  class="pb-3 block bg-clip-text text-transparent bg-gradient-to-r from-teal-200 to-cyan-400 sm:pb-5"
                >
                  ship web apps
                </span>
              </h1>
              <p
                class="text-base text-gray-300 sm:text-xl lg:text-lg xl:text-xl"
              >
                Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui Lorem cupidatat commodo. Elit sunt amet fugiat veniam occaecat fugiat.
              </p>
              <div
                class="mt-10 sm:mt-12"
              >
                <form
                  action="#"
                  class="sm:max-w-xl sm:mx-auto lg:mx-0"
                >
                  <div
                    class="sm:flex"
                  >
                    <div
                      class="min-w-0 flex-1"
                    >
                      <label
                        class="sr-only"
                        for="email"
                      >
                        Email address
                      </label>
                      <input
                        class="block w-full px-4 py-3 rounded-md border-0 text-base text-gray-900 placeholder-gray-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-cyan-400 focus:ring-offset-gray-900"
                        id="email"
                        placeholder="Enter your email"
                        type="email"
                      />
                    </div>
                    <div
                      class="mt-3 sm:mt-0 sm:ml-3"
                    >
                      <button
                        class="block w-full py-3 px-4 rounded-md shadow bg-gradient-to-r from-teal-500 to-cyan-600 text-white font-medium hover:from-teal-600 hover:to-cyan-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-cyan-400 focus:ring-offset-gray-900"
                        type="submit"
                      >
                        Start free trial
                      </button>
                    </div>
                  </div>
                  <p
                    class="mt-3 text-sm text-gray-300 sm:mt-4"
                  >
                    Start your free 14-day trial, no credit card necessary. By providing your email, you agree to our
                     
                    <a
                      class="font-medium text-white"
                      href="#"
                    >
                      terms of service
                    </a>
                    .
                  </p>
                </form>
              </div>
            </div>
          </div>
          <div
            class="mt-12 -mb-16 sm:-mb-48 lg:m-0 lg:relative"
          >
            <div
              class="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 lg:max-w-none lg:px-0"
            >
              <img
                alt=""
                class="w-full lg:absolute lg:inset-y-0 lg:left-0 lg:h-full lg:w-auto lg:max-w-none"
                src="https://tailwindui.com/img/component-images/cloud-illustration-teal-cyan.svg"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="relative pt-16 sm:pt-24 lg:pt-32"
    >
      <div
        class="mx-auto max-w-md px-4 text-center sm:px-6 sm:max-w-3xl lg:px-8 lg:max-w-7xl"
      >
        <div>
          <h2
            class="text-base font-semibold tracking-wider text-accent uppercase"
          >
            Serverless
          </h2>
          <p
            class="mt-2 text-3xl font-extrabold text-base-content tracking-tight sm:text-4xl"
          >
            No server? No problem.
          </p>
          <p
            class="mt-5 max-w-prose mx-auto text-xl text-gray-500"
          >
            Phasellus lorem quam molestie id quisque diam aenean nulla in. Accumsan in quis quis nunc, ullamcorper malesuada. Eleifend condimentum id viverra nulla.
          </p>
        </div>
        <div
          class="mt-12 -mb-10 sm:-mb-24 lg:-mb-80"
        >
          <img
            alt=""
            class="rounded-lg shadow-xl ring-1 ring-black ring-opacity-5"
            src="https://tailwindui.com/img/component-images/green-project-app-screenshot.jpg"
          />
        </div>
      </div>
    </div>
    <div
      class="relative bg-base py-16 sm:py-24 lg:py-32"
    >
      <div
        class="mx-auto max-w-md px-4 text-center sm:max-w-3xl sm:px-6 lg:px-8 lg:max-w-7xl"
      >
        <h2
          class="text-base font-semibold tracking-wider text-cyan-600 uppercase"
        >
          Deploy faster
        </h2>
        <p
          class="mt-2 text-3xl font-extrabold text-base-content tracking-tight sm:text-4xl"
        >
          Everything you need to deploy your app
        </p>
        <p
          class="mt-5 max-w-prose mx-auto text-xl text-gray-500"
        >
          Phasellus lorem quam molestie id quisque diam aenean nulla in. Accumsan in quis quis nunc, ullamcorper malesuada. Eleifend condimentum id viverra nulla.
        </p>
        <div
          class="mt-12"
        >
          <div
            class="grid grid-cols-1 gap-8 sm:grid-cols-2 lg:grid-cols-3"
          >
            <div
              class="pt-6"
            >
              <div
                class="flow-root bg-gray-50 rounded-lg px-6 pb-8"
              >
                <div
                  class="-mt-6"
                >
                  <div>
                    <span
                      class="inline-flex items-center justify-center p-3 bg-gradient-to-r from-teal-500 to-cyan-600 rounded-md shadow-lg"
                    >
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6 text-white"
                        viewBox="0 0 18 18"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          class="a"
                          d="M14.643 4.736a4.394 4.394 0 0 0-8.51-1.521 3.861 3.861 0 0 0-3.844 3.7A2.612 2.612 0 0 0 .516 9.687 2.673 2.673 0 0 0 3.205 12H5.75a.25.25 0 0 0 .25-.25v-.5a.25.25 0 0 0-.25-.25H3.2a1.668 1.668 0 0 1-1.696-1.52 1.607 1.607 0 0 1 1.518-1.692l.087-.003h.18v-.714a2.86 2.86 0 0 1 3.6-2.759 3.393 3.393 0 1 1 6.634 1.35 2.679 2.679 0 1 1 .3 5.34H12.25a.25.25 0 0 0-.25.25v.5a.25.25 0 0 0 .25.25h1.45a3.759 3.759 0 0 0 3.774-3.242 3.684 3.684 0 0 0-2.831-4.024Z"
                          style="fill: hsl(var(--bc));"
                        />
                        <path
                          class="a"
                          d="M6.75 9H8v7.5a.5.5 0 0 0 .5.5h1a.5.5 0 0 0 .5-.5V9h1.25a.25.25 0 0 0 .25-.25.245.245 0 0 0-.059-.159L9.182 6.08a.25.25 0 0 0-.353-.01l-.011.01-2.26 2.51a.245.245 0 0 0-.058.16.25.25 0 0 0 .25.25Z"
                          style="fill: hsl(var(--bc));"
                        />
                      </svg>
                    </span>
                  </div>
                  <h3
                    class="mt-8 text-lg font-medium text-gray-900 tracking-tight"
                  >
                    Push to Deploy
                  </h3>
                  <p
                    class="mt-5 text-base text-gray-500"
                  >
                    Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi vitae lobortis.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="pt-6"
            >
              <div
                class="flow-root bg-gray-50 rounded-lg px-6 pb-8"
              >
                <div
                  class="-mt-6"
                >
                  <div>
                    <span
                      class="inline-flex items-center justify-center p-3 bg-gradient-to-r from-teal-500 to-cyan-600 rounded-md shadow-lg"
                    >
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6 text-white"
                        viewBox="0 0 18 18"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <title>
                          S LockClosed 18 N
                        </title>
                        <path
                          d="M14.5 8H14V7A5 5 0 0 0 4 7v1h-.5a.5.5 0 0 0-.5.5v8a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 .5-.5v-8a.5.5 0 0 0-.5-.5ZM6 7a3 3 0 0 1 6 0v1H6Zm4 6.111V14.5a.5.5 0 0 1-.5.5h-1a.5.5 0 0 1-.5-.5v-1.389a1.5 1.5 0 1 1 2 0Z"
                          style="fill: hsl(var(--bc));"
                        />
                      </svg>
                    </span>
                  </div>
                  <h3
                    class="mt-8 text-lg font-medium text-gray-900 tracking-tight"
                  >
                    SSL Certificates
                  </h3>
                  <p
                    class="mt-5 text-base text-gray-500"
                  >
                    Qui aut temporibus nesciunt vitae dicta repellat sit dolores pariatur. Temporibus qui illum aut.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="pt-6"
            >
              <div
                class="flow-root bg-gray-50 rounded-lg px-6 pb-8"
              >
                <div
                  class="-mt-6"
                >
                  <div>
                    <span
                      class="inline-flex items-center justify-center p-3 bg-gradient-to-r from-teal-500 to-cyan-600 rounded-md shadow-lg"
                    >
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6 text-white"
                        viewBox="0 0 18 18"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <title>
                          S Refresh 18 N
                        </title>
                        <path
                          class="a"
                          d="M16.337 10h-.947a.608.608 0 0 0-.581.469 5.723 5.723 0 0 1-9.559 2.537l-.346-.346 1.978-1.978A.392.392 0 0 0 7 10.4a.4.4 0 0 0-.377-.4H1.25a.25.25 0 0 0-.25.25v5.375a.4.4 0 0 0 .4.375.39.39 0 0 0 .28-.118l1.808-1.808.179.178a8.09 8.09 0 0 0 3.642 2.165 7.715 7.715 0 0 0 9.437-5.474q.048-.178.087-.358a.5.5 0 0 0-.496-.585ZM16.6 2a.39.39 0 0 0-.28.118l-1.81 1.808-.179-.176a8.09 8.09 0 0 0-3.642-2.166 7.715 7.715 0 0 0-9.436 5.477q-.047.176-.086.354A.5.5 0 0 0 1.663 8h.947a.608.608 0 0 0 .581-.469 5.723 5.723 0 0 1 9.559-2.537l.346.346-1.978 1.978A.392.392 0 0 0 11 7.6a.4.4 0 0 0 .377.4h5.373a.25.25 0 0 0 .25-.25V2.377A.4.4 0 0 0 16.6 2Z"
                          style="fill: hsl(var(--bc));"
                        />
                      </svg>
                    </span>
                  </div>
                  <h3
                    class="mt-8 text-lg font-medium text-gray-900 tracking-tight"
                  >
                    Simple Queues
                  </h3>
                  <p
                    class="mt-5 text-base text-gray-500"
                  >
                    Rerum quas incidunt deleniti quaerat suscipit mollitia. Amet repellendus ut odit dolores qui.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="pt-6"
            >
              <div
                class="flow-root bg-gray-50 rounded-lg px-6 pb-8"
              >
                <div
                  class="-mt-6"
                >
                  <div>
                    <span
                      class="inline-flex items-center justify-center p-3 bg-gradient-to-r from-teal-500 to-cyan-600 rounded-md shadow-lg"
                    >
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6 text-white"
                        viewBox="0 0 18 18"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <title>
                          S Shield 18 N
                        </title>
                        <path
                          d="M15 1.5a.5.5 0 0 0-.5-.5h-11a.5.5 0 0 0-.5.5v6.55a7.804 7.804 0 0 0 2.929 6.093l2.836 2.178a.35.35 0 0 0 .469 0l2.837-2.178A7.804 7.804 0 0 0 15 8.05ZM4.861 11.143A7.24 7.24 0 0 1 4 8V2h10Z"
                          style="fill: hsl(var(--bc));"
                        />
                      </svg>
                    </span>
                  </div>
                  <h3
                    class="mt-8 text-lg font-medium text-gray-900 tracking-tight"
                  >
                    Advanced Security
                  </h3>
                  <p
                    class="mt-5 text-base text-gray-500"
                  >
                    Ullam laboriosam est voluptatem maxime ut mollitia commodi. Et dignissimos suscipit perspiciatis.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="pt-6"
            >
              <div
                class="flow-root bg-gray-50 rounded-lg px-6 pb-8"
              >
                <div
                  class="-mt-6"
                >
                  <div>
                    <span
                      class="inline-flex items-center justify-center p-3 bg-gradient-to-r from-teal-500 to-cyan-600 rounded-md shadow-lg"
                    >
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6 text-white"
                        viewBox="0 0 18 18"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          class="a"
                          d="M8.597 12.893H7.534a3.07 3.07 0 0 0-.45-1.09l.756-.756a.303.303 0 0 0 0-.429l-.46-.46a.303.303 0 0 0-.429 0l-.755.757a3.073 3.073 0 0 0-1.089-.45V9.403a.303.303 0 0 0-.304-.303h-.607a.303.303 0 0 0-.303.303v1.063a3.073 3.073 0 0 0-1.089.45l-.757-.758a.303.303 0 0 0-.429 0l-.46.46a.303.303 0 0 0 0 .429l.757.757a3.07 3.07 0 0 0-.45 1.09H.403a.303.303 0 0 0-.303.303v.607a.303.303 0 0 0 .303.303h1.063a3.07 3.07 0 0 0 .45 1.09l-.758.756a.303.303 0 0 0 0 .429l.46.46a.303.303 0 0 0 .429 0l.757-.757a3.073 3.073 0 0 0 1.089.45v1.062a.303.303 0 0 0 .303.304h.607a.303.303 0 0 0 .304-.304v-1.063a3.073 3.073 0 0 0 1.089-.45l.756.756a.303.303 0 0 0 .43 0l.46-.46a.303.303 0 0 0 0-.429l-.757-.755a3.07 3.07 0 0 0 .45-1.09h1.063a.303.303 0 0 0 .303-.303v-.607a.303.303 0 0 0-.303-.303ZM4.5 15.082A1.582 1.582 0 1 1 6.082 13.5 1.582 1.582 0 0 1 4.5 15.082ZM17.681 7.453l-1.4-.572a4.378 4.378 0 0 0-.006-1.678l1.405-.591a.432.432 0 0 0 .231-.566l-.361-.855a.432.432 0 0 0-.565-.23h-.001l-1.406.589a4.38 4.38 0 0 0-1.195-1.178l.572-1.4a.432.432 0 0 0-.238-.564l-.8-.326a.432.432 0 0 0-.563.237l-.572 1.4a4.38 4.38 0 0 0-1.678.006L10.512.322a.432.432 0 0 0-.566-.231L9.092.45a.432.432 0 0 0-.232.566l.59 1.405a4.377 4.377 0 0 0-1.178 1.197l-1.4-.572a.432.432 0 0 0-.564.237l-.326.8a.432.432 0 0 0 .237.564l1.4.571a4.378 4.378 0 0 0 .006 1.679l-1.405.59a.432.432 0 0 0-.231.567l.36.854a.432.432 0 0 0 .565.23l1.405-.59a4.38 4.38 0 0 0 1.196 1.178l-.571 1.4a.432.432 0 0 0 .237.563l.8.327a.432.432 0 0 0 .563-.237l.572-1.4a4.378 4.378 0 0 0 1.678-.006l.591 1.405a.432.432 0 0 0 .566.23l.855-.359a.432.432 0 0 0 .231-.566l-.587-1.405a4.376 4.376 0 0 0 1.178-1.195l1.4.572a.432.432 0 0 0 .563-.237l.327-.8a.432.432 0 0 0-.236-.565Zm-5.731.691a2.094 2.094 0 1 1 2.094-2.094 2.094 2.094 0 0 1-2.094 2.094Z"
                          style="fill: hsl(var(--bc));"
                        />
                      </svg>
                    </span>
                  </div>
                  <h3
                    class="mt-8 text-lg font-medium text-gray-900 tracking-tight"
                  >
                    Powerful API
                  </h3>
                  <p
                    class="mt-5 text-base text-gray-500"
                  >
                    Ab a facere voluptatem in quia corrupti veritatis aliquam. Veritatis labore quaerat ipsum quaerat id.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="pt-6"
            >
              <div
                class="flow-root bg-gray-50 rounded-lg px-6 pb-8"
              >
                <div
                  class="-mt-6"
                >
                  <div>
                    <span
                      class="inline-flex items-center justify-center p-3 bg-gradient-to-r from-teal-500 to-cyan-600 rounded-md shadow-lg"
                    >
                      <svg
                        aria-hidden="true"
                        class="h-6 w-6 text-white"
                        viewBox="0 0 18 18"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M5.5 5h11a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-11a.5.5 0 0 0-.5.5V3H2V1.25A.25.25 0 0 0 1.75 1h-.5a.25.25 0 0 0-.25.25v15.5a.25.25 0 0 0 .25.25h.5a.25.25 0 0 0 .25-.25V15h3v1.5a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-11a.5.5 0 0 0-.5.5v.5H2v-4h3v.5a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-11a.5.5 0 0 0-.5.5V9H2V4h3v.5a.5.5 0 0 0 .5.5Zm.5 9h2v1H6Zm0-6h2v1H6Zm0-6h2v1H6Z"
                          style="fill: hsl(var(--bc));"
                        />
                      </svg>
                    </span>
                  </div>
                  <h3
                    class="mt-8 text-lg font-medium text-gray-900 tracking-tight"
                  >
                    Database Backups
                  </h3>
                  <p
                    class="mt-5 text-base text-gray-500"
                  >
                    Quia qui et est officia cupiditate qui consectetur. Ratione similique et impedit ea ipsum et.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="relative py-16 sm:py-24 lg:py-32"
    >
      <div
        class="relative"
      >
        <div
          class="text-center mx-auto max-w-md px-4 sm:max-w-3xl sm:px-6 lg:px-8 lg:max-w-7xl"
        >
          <h2
            class="text-base font-semibold tracking-wider text-cyan-600 uppercase"
          >
            Learn
          </h2>
          <p
            class="mt-2 text-3xl font-extrabold text-base-content tracking-tight sm:text-4xl"
          >
            Helpful Resources
          </p>
          <p
            class="mt-5 mx-auto max-w-prose text-xl text-gray-500"
          >
            Phasellus lorem quam molestie id quisque diam aenean nulla in. Accumsan in quis quis nunc, ullamcorper malesuada. Eleifend condimentum id viverra nulla.
          </p>
        </div>
        <div
          class="mt-12 mx-auto max-w-md px-4 grid gap-8 sm:max-w-lg sm:px-6 lg:px-8 lg:grid-cols-3 lg:max-w-7xl"
        >
          <div
            class="flex flex-col rounded-lg shadow-lg overflow-hidden"
          >
            <div
              class="flex-shrink-0"
            >
              <img
                alt=""
                class="h-48 w-full object-cover"
                src="https://images.unsplash.com/photo-1496128858413-b36217c2ce36?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1679&q=80"
              />
            </div>
            <div
              class="flex-1 bg-white p-6 flex flex-col justify-between"
            >
              <div
                class="flex-1"
              >
                <p
                  class="text-sm font-medium text-cyan-600"
                >
                  <a
                    class="hover:underline"
                    href="#"
                  >
                    Article
                  </a>
                </p>
                <a
                  class="block mt-2"
                  href="#"
                >
                  <p
                    class="text-xl font-semibold text-gray-900"
                  >
                    Boost your conversion rate
                  </p>
                  <p
                    class="mt-3 text-base text-gray-500"
                  >
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto accusantium praesentium eius, ut atque fuga culpa, similique sequi cum eos quis dolorum.
                  </p>
                </a>
              </div>
              <div
                class="mt-6 flex items-center"
              >
                <div
                  class="flex-shrink-0"
                >
                  <a
                    href="#"
                  >
                    <img
                      alt="Roel Aufderehar"
                      class="h-10 w-10 rounded-full"
                      src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
                    />
                  </a>
                </div>
                <div
                  class="ml-3"
                >
                  <p
                    class="text-sm font-medium text-gray-900"
                  >
                    <a
                      class="hover:underline"
                      href="#"
                    >
                      Roel Aufderehar
                    </a>
                  </p>
                  <div
                    class="flex space-x-1 text-sm text-gray-500"
                  >
                    <time
                      datetime="2020-03-16"
                    >
                      Mar 16, 2020
                    </time>
                    <span
                      aria-hidden="true"
                    >
                      ·
                    </span>
                    <span>
                      6 min
                       read
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col rounded-lg shadow-lg overflow-hidden"
          >
            <div
              class="flex-shrink-0"
            >
              <img
                alt=""
                class="h-48 w-full object-cover"
                src="https://images.unsplash.com/photo-1547586696-ea22b4d4235d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1679&q=80"
              />
            </div>
            <div
              class="flex-1 bg-white p-6 flex flex-col justify-between"
            >
              <div
                class="flex-1"
              >
                <p
                  class="text-sm font-medium text-cyan-600"
                >
                  <a
                    class="hover:underline"
                    href="#"
                  >
                    Video
                  </a>
                </p>
                <a
                  class="block mt-2"
                  href="#"
                >
                  <p
                    class="text-xl font-semibold text-gray-900"
                  >
                    How to use search engine optimization to drive sales
                  </p>
                  <p
                    class="mt-3 text-base text-gray-500"
                  >
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit facilis asperiores porro quaerat doloribus, eveniet dolore. Adipisci tempora aut inventore optio animi., tempore temporibus quo laudantium.
                  </p>
                </a>
              </div>
              <div
                class="mt-6 flex items-center"
              >
                <div
                  class="flex-shrink-0"
                >
                  <a
                    href="#"
                  >
                    <img
                      alt="Brenna Goyette"
                      class="h-10 w-10 rounded-full"
                      src="https://images.unsplash.com/photo-1550525811-e5869dd03032?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
                    />
                  </a>
                </div>
                <div
                  class="ml-3"
                >
                  <p
                    class="text-sm font-medium text-gray-900"
                  >
                    <a
                      class="hover:underline"
                      href="#"
                    >
                      Brenna Goyette
                    </a>
                  </p>
                  <div
                    class="flex space-x-1 text-sm text-gray-500"
                  >
                    <time
                      datetime="2020-03-10"
                    >
                      Mar 10, 2020
                    </time>
                    <span
                      aria-hidden="true"
                    >
                      ·
                    </span>
                    <span>
                      4 min
                       read
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col rounded-lg shadow-lg overflow-hidden"
          >
            <div
              class="flex-shrink-0"
            >
              <img
                alt=""
                class="h-48 w-full object-cover"
                src="https://images.unsplash.com/photo-1492724441997-5dc865305da7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1679&q=80"
              />
            </div>
            <div
              class="flex-1 bg-white p-6 flex flex-col justify-between"
            >
              <div
                class="flex-1"
              >
                <p
                  class="text-sm font-medium text-cyan-600"
                >
                  <a
                    class="hover:underline"
                    href="#"
                  >
                    Case Study
                  </a>
                </p>
                <a
                  class="block mt-2"
                  href="#"
                >
                  <p
                    class="text-xl font-semibold text-gray-900"
                  >
                    Improve your customer experience
                  </p>
                  <p
                    class="mt-3 text-base text-gray-500"
                  >
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint harum rerum voluptatem quo recusandae magni placeat saepe molestiae, sed excepturi cumque corporis perferendis hic.
                  </p>
                </a>
              </div>
              <div
                class="mt-6 flex items-center"
              >
                <div
                  class="flex-shrink-0"
                >
                  <a
                    href="#"
                  >
                    <img
                      alt="Daniela Metz"
                      class="h-10 w-10 rounded-full"
                      src="https://images.unsplash.com/photo-1487412720507-e7ab37603c6f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
                    />
                  </a>
                </div>
                <div
                  class="ml-3"
                >
                  <p
                    class="text-sm font-medium text-gray-900"
                  >
                    <a
                      class="hover:underline"
                      href="#"
                    >
                      Daniela Metz
                    </a>
                  </p>
                  <div
                    class="flex space-x-1 text-sm text-gray-500"
                  >
                    <time
                      datetime="2020-02-12"
                    >
                      Feb 12, 2020
                    </time>
                    <span
                      aria-hidden="true"
                    >
                      ·
                    </span>
                    <span>
                      11 min
                       read
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="relative bg-gray-900"
    >
      <div
        class="relative h-56 bg-accent sm:h-72 md:absolute md:left-0 md:h-full md:w-1/2"
      >
        <img
          alt=""
          class="w-full h-full object-cover"
          src="https://images.unsplash.com/photo-1525130413817-d45c1d127c42?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1920&q=60&sat=-100"
        />
        <div
          aria-hidden="true"
          class="absolute inset-0 bg-gradient-to-r from-teal-500 to-cyan-600 mix-blend-multiply"
        />
      </div>
      <div
        class="relative mx-auto max-w-md px-4 py-12 sm:max-w-7xl sm:px-6 sm:py-20 md:py-28 lg:px-8 lg:py-32"
      >
        <div
          class="md:ml-auto md:w-1/2 md:pl-10"
        >
          <h2
            class="text-base font-semibold uppercase tracking-wider text-gray-300"
          >
            Award winning support
          </h2>
          <p
            class="mt-2 text-white text-3xl font-extrabold tracking-tight sm:text-4xl"
          >
            We’re here to help
          </p>
          <p
            class="mt-3 text-lg text-gray-300"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Et, egestas tempus tellus etiam sed. Quam a scelerisque amet ullamcorper eu enim et fermentum, augue. Aliquet amet volutpat quisque ut interdum tincidunt duis.
          </p>
          <div
            class="mt-8"
          >
            <div
              class="inline-flex rounded-md shadow"
            >
              <a
                class="inline-flex items-center justify-center px-5 py-3 border border-transparent text-base font-medium rounded-md text-gray-900 bg-white hover:bg-gray-50"
                href="#"
              >
                Visit the help center
                <svg
                  aria-hidden="true"
                  class="-mr-1 ml-3 h-5 w-5 text-gray-400"
                  viewBox="0 0 18 18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="a"
                    d="M16.5 9h-1a.5.5 0 0 0-.5.5V15H3V3h5.5a.5.5 0 0 0 .5-.5v-1a.5.5 0 0 0-.5-.5h-7a.5.5 0 0 0-.5.5v15a.5.5 0 0 0 .5.5h15a.5.5 0 0 0 .5-.5v-7a.5.5 0 0 0-.5-.5Z"
                    style="fill: hsl(var(--bc));"
                  />
                  <path
                    class="a"
                    d="M16.75 1h-5.373a.4.4 0 0 0-.377.4.392.392 0 0 0 .117.28l1.893 1.895-3.52 3.521a.5.5 0 0 0 0 .707l.706.708a.5.5 0 0 0 .708 0l3.521-3.521 1.893 1.892A.39.39 0 0 0 16.6 7a.4.4 0 0 0 .4-.377V1.25a.25.25 0 0 0-.25-.25Z"
                    style="fill: hsl(var(--bc));"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
